### DISCUSSION TITLE
Why are there so much random icons on ionicons

### DISCUSSION THREAD

**Discussion by fergassss1** [fast4x/RiMusic#4202] [General]
On https://github.com/fast4x/RiMusic/tree/master/assets/ionicons there's like a gazillion random icons.
For example this random restaurant icon 😭https://github.com/fast4x/RiMusic/blob/master/assets/ionicons/restaurant.svg

**Comment by twistios** [comment]
Ionicons has many different icons and RiMusic simply has a copy with all of them.
